AppController/djinn.rb

```diff
@@ -711,11 +711,34 @@ def kill(stop_deployment, secret)
           end
         }
       }
-      Djinn.log_info('Waiting for other nodes to acknoledge stop ... ')
-      threads.each { |t| t.join }
     end
 
-    Djinn.log_info('---- Stopping AppController ----')
+    Djinn.log_devug("Stopping local services ...")
```

Comment:
I think we should settle on a single method of stopping services. If both the controller and the terminate script have a similar method of stopping services, we need to keep both in sync. In addition, we would need to make sure they both aren't trying to stop stuff at the same time since that can confuse monit.

To me, it would make sense if sigterm just stopped the controller process rather than all the services on the machine.